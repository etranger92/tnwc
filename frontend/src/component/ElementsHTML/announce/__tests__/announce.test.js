import * as React from 'react';
import AnnounceOne from "../Announce.tsx";
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';

describe("We test the innerHTML of the AnnounceOne", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = mount(<AnnounceOne />);
    });
  test("Snapshot AnnounceOne ", () => {
    //It works
 expect(toJson(wrapper)).toMatchSnapshot();
  });
});
