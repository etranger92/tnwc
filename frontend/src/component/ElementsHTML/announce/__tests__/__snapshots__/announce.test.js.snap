// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`We test the innerHTML of the AnnounceOne Snapshot AnnounceOne  1`] = `
<AnnounceOne>
  <section
    id="announce-one"
  >
    <article>
      <div
        id="sub-announce-one"
      >
        <h4
          className=""
        >
          Friday 16th October 2020 at 10am
        </h4>
        <h1
          className=""
        >
          Virtual open day
        </h1>
      </div>
      <div
        className=""
        id="sub-announce-two"
      >
        <div
          className="sub_announce_two"
        >
          <h4>
            First part
          </h4>
          <div
            className="sub_announce_two_paragraph"
          >
            <p>
              Introduction to education and training
            </p>
            <p>
              Certificate in education and training
            </p>
            <p>
              Diploma in education and training
            </p>
          </div>
        </div>
        <div
          className="sub_announce_two"
        >
           
          <h4>
             Second part 
          </h4>
          <div
            className="sub_announce_two_paragraph"
          >
            <p>
              Top-up degrees/spedification
            </p>
            <p>
              Qtls
            </p>
          </div>
        </div>
        <div
          className="sub_announce_two"
        >
           
          <h4>
             Third part 
          </h4>
          <div
            className="sub_announce_two_paragraph"
          >
            <p>
              Questions
            </p>
          </div>
        </div>
      </div>
    </article>
  </section>
</AnnounceOne>
`;
